src/CloudFoundry.Build.Tasks.IntegrationTests/LoadYamlTest.cs

```diff
@@ -3,18 +3,20 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Reflection;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace CloudFoundry.Build.Tasks.IntegrationTests
 {
     [TestClass]
+    [Ignore]
```

Comment:
(Talked offline -- yaml loading feature isn't working at the moment. This is being tracked separately).